utilities/transactions/pessimistic_transaction_db.h

```diff
@@ -179,6 +185,13 @@ class WritePreparedTxnDB : public PessimisticTransactionDB {
   void AddCommitted(uint64_t prepare_seq, uint64_t commit_seq);
 
  private:
+  friend class WritePreparedTransactionTest_IsInSnapshotTest_Test;
+
+  void init(const TransactionDBOptions& /* unused */) {
+    commit_cache_ =
+        unique_ptr<CommitEntry[]>(new CommitEntry[COMMIT_CACHE_SIZE]{});
```

Comment:
I'm seeing this error when building myrocks:
https://gist.github.com/yiwu-arbug/eb1d8a1f037400d6ccbb204d916eb9b0